src/DropdownMenu/index.ts

```diff
@@ -0,0 +1,3 @@
+export {DropdownMenu} from './DropdownMenu'
+export type {DropdownMenuProps} from './DropdownMenu'
+export {DropdownButton} from './DropdownButton'
```

Comment:
Should we export `DropdownMenuButton` props as well?

Reply:
Sounds like a good idea.  Done in https://github.com/primer/components/pull/1135/commits/edf01d5694dbb3df8a7336a156ba45ec241836e9